include/PianoRoll.h

```diff
@@ -247,13 +252,17 @@ protected slots:
 private:
 	enum Actions
 	{
-		ActionNone,
-		ActionMoveNote,
-		ActionResizeNote,
-		ActionSelectNotes,
-		ActionChangeNoteProperty,
-		ActionResizeNoteEditArea,
-		ActionKnife
+		ActionNone                          = 0,
```

Comment:
We should probably update this to an enum class. https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#enum3-prefer-class-enums-over-plain-enums